mmv1/products/alloydb/Backup.yaml

```diff
@@ -23,26 +23,13 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
```

Comment:
You seem to have picked up a bad diff here!

Reply:
This was actually intentional - I wanted to make sure that I was correct that we could simplify this block. Looks like I was wrong 😭

Reply:
fixed in de864b92d